local-interchain/cmd/local-ic/start_chain.go

```diff
@@ -94,6 +97,27 @@ local-ic start https://pastebin.com/raw/Ummk4DTM
 	},
 }
 
+// GetConfigWithExtension returns the config with the file extension attached if one was not provided.
+// If "hub" is passed it, it will search for hub.yaml, hub.yml, or hub.json.
+// If an extension is already applied, it will use that.
+func GetConfigWithExtension(parentDir, config string) (string, error) {
```

Comment:
before I just appended .json to the end of the filename if it was not found, instead it's better to try and find the file with possible extensions, then return that instead.